### DISCUSSION TITLE
New SDL fixes Xbox Controller reconnecting issue, but..

### DISCUSSION THREAD

**Discussion by koga-ninja** [AntiMicroX/antimicrox#807] [General]
When updating the SDL file for AntimicroX, commands assigned to the left and right triggers do not work unless another input is held down simultaneously. I have added this as a discussion since AntimicroX does not currently come with the newest SDL release, but I wanted to bring awareness to the issue. Thanks for all the great work to the development team as always..

**Comment by koga-ninja** [comment]
Issue likely fixed(haven't tested):

https://github.com/libsdl-org/SDL/issues/7940#issuecomment-1625683135

**Reply by pktiuk** [reply]
@koga-ninja   
Which release of the SDL will contain this fix?

**Reply by koga-ninja** [reply]
Was told next release. 2.28.2. 

https://github.com/libsdl-org/SDL/issues/7940#issuecomment-1633395051

**Reply by pktiuk** [reply]
Later before the next release of the AntiMicroX I will update SDL version bundled with Windows and AppImage packages (in case of classical packages SDL version depends on your distro).
